.github/dependabot.yml

```diff
@@ -5,6 +5,11 @@ updates:
     schedule:
       interval: "daily"
     open-pull-requests-limit: 10
+    ignore:
+      - dependency-name: "mock-jwks"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "@types/node"
```

Comment:
@jasonaowen wanted to tag you here in case there's a way to undo https://github.com/PhilanthropyDataCommons/service/pull/370 now that we have the "right" way of stopping major node updates

Reply:
The documentation in #370's description says

> `@dependabot ignore this major version` will close this PR and stop Dependabot creating any more for this major version (unless you reopen the PR or upgrade to it yourself)

So, we can reopen the PR if you want, but I don't think it's a problem to have both an explicit rule and also a one-off comment directive telling Dependabot not to try to open 18->20 PRs.

Reply:
The problem arises if we ever update to Node 20 -- at which point we would want dependabot updates and the only way to re-enable them would be to sift through 5 years worth of dependabot prs to find the right one 😅